src/sage/combinat/designs/covering_array.py

```diff
@@ -0,0 +1,43 @@
+r"""
+Covering Arrays (CA)
+
+A Covering Array, denoted CA`(N,k,v,t)`, is an `n` by `k` array with
+entries from a set of `v` elements with the property that in every
+selection of `t` columns, eachrow contains every sequence of
+`t`-elements at least once.
+
+An Orthogonal Array, denoted OA`(N,k,v,t)` is a covering array with the
+property that each row contains every sequence of `t`-elements exactly
+once
+
+REFERENCES:
+
+- [Colb2004]_
+
+- [Sher2006]_
+
+- [Wal2007]_
+
+AUTHORS:
+
+- Aaron Dwyer and brett stevens (2022): initial version
+
+.. TODO::
+
+    Implement various construction methods for CAs
+
+"""
```

Comment:
It feels like the module documentation is lacking examples. You should have a look at `orthogonal_arrays.py` for inspiration.